projects/snapshot_bot/config.h

```diff
@@ -39,7 +39,9 @@ class Config
     bool shouldUseInfoMax() const{ return m_UseInfoMax; }
     bool shouldSaveTestingDiagnostic() const{ return m_SaveTestingDiagnostic; }
     bool shouldStreamOutput() const{ return m_StreamOutput; }
-
+    bool shouldUseNorbot() const{ return m_UseNorbot; }
+    bool shouldUseMecanum() const{ return m_UseMecanum; }
```

Comment:
You can already set the type of tank robot you're using at compile time by either setting the ``TANK_TYPE`` env var or adding e.g. ``-DTANK_TYPE=EV3`` to the CMake args. I've been fiddling with that a bit on another branch, so I could add a similar mechanism to include Mecanum robots if you like?

Reply:
Sure! It doesn't exactly make a lot of sense to have this configured at runtime anyway...